src/assets/i18n/en.json5

```diff
@@ -4839,4 +4839,8 @@
   "home.recent-submissions.head": "Recent Submissions",
 
   "listable-notification-object.default-message": "This object couldn't be retrieved",
+
+  "search.filters.filter.show-tree": "Browse {{ name }} tree",
+
+  "vocabulary-treeview.info": "Select a subject to add as search filter"
```

Comment:
Same here, move next to the related keys (`vocabulary-treeview.*`) around line 4501.  Small grammar change, this should likely say: "Select a subject to add as a search filter"

Reply:
Resolved here https://github.com/DSpace/dspace-angular/commit/e0cadec2c21b44587945edd08e17c3fda1532ed0